test/test_sort.py

```diff
@@ -0,0 +1,107 @@
+
+            resp = self.server.workbooks._make_request(requests.get,
+                                                       url,
+                                                       content=None,
+                                                       request_object=opts,
+                                                       auth_token='j80k54ll2lfMZ0tv97mlPvvSCRyD0DOM',
+                                                       content_type='text/xml')
+
+            self.assertEqual(resp.request.query, 'pagenumber=13&pagesize=13&filter=name:eq:superstore')
+
+    def test_filter_equals_list(self):
+        with self.assertRaises(ValueError) as cm:
+            TSC.Filter(TSC.RequestOptions.Field.Tags,
+                       TSC.RequestOptions.Operator.Equals,
+                       ['foo', 'bar'])
+
+        self.assertEquals("Filter values can only be a list if the operator is 'in'.", str(cm.exception)),
+
+    def test_filter_in(self):
+        with requests_mock.mock() as m:
+            m.get(requests_mock.ANY)
+            url = "http://test/api/2.3/sites/dad65087-b08b-4603-af4e-2887b8aafc67/workbooks"
+            opts = TSC.RequestOptions(pagesize=13, pagenumber=13)
+
+            opts.filter.add(TSC.Filter(TSC.RequestOptions.Field.Tags,
+                                       TSC.RequestOptions.Operator.In,
+                                       ['stocks', 'market']))
+
+            resp = self.server.workbooks._make_request(requests.get,
+                                                       url,
+                                                       content=None,
+                                                       request_object=opts,
+                                                       auth_token='j80k54ll2lfMZ0tv97mlPvvSCRyD0DOM',
+                                                       content_type='text/xml')
+
+            self.assertEqual(resp.request.query, 'pagenumber=13&pagesize=13&filter=tags:in:[stocks,market]')
+
+    def test_sort_asc(self):
+        with requests_mock.mock() as m:
+            m.get(requests_mock.ANY)
+            url = "http://test/api/2.3/sites/dad65087-b08b-4603-af4e-2887b8aafc67/workbooks"
+            opts = TSC.RequestOptions(pagesize=13, pagenumber=13)
+            opts.sort.add(TSC.Sort(TSC.RequestOptions.Field.Name,
+                                   TSC.RequestOptions.Direction.Asc))
+
+            resp = self.server.workbooks._make_request(requests.get,
+                                                       url,
+                                                       content=None,
+                                                       request_object=opts,
+                                                       auth_token='j80k54ll2lfMZ0tv97mlPvvSCRyD0DOM',
+                                                       content_type='text/xml')
+
+            self.assertEqual(resp.request.query, 'pagenumber=13&pagesize=13&sort=name:asc')
+
+    # def test_filter_combo(self):
```

Comment:
This is because of the determinism problem, right?

After @RussTheAerialist does his bit uncomment and rebase on top of his changes and this should be good to go.